.github/workflows/publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Publish wheel and source distributions to PyPI
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.12
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.12
+    - name: Build source distribution
+      run: |
+        python -m pip install build
+        python -m build
+    - name: Store them as artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: sdist
+        path: dist/*.tar.gz
+
+  build_wheel:
+    name: Build wheel
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.12
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.12
+    - name: Build distributions
+      run: |
+        python -m pip install build
+        python -m build --wheel
+    - name: Store them as artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: wheel
+        path: dist/*.whl
```

Comment:
Why no cibuildwheel?

Reply:
Not required since MultiscaleRun is pure Python.